web/template/manage-deputy-contact-details.gotmpl

```diff
@@ -0,0 +1,227 @@
+{{ template "page" . }}
+{{ define "main" }}
+    {{ template "pro-deputy" . }}
+    {{ template "navigation" . }}
+    {{ template "error-summary" .Errors }}
+
+
+    <div class="govuk-grid-row govuk-!-margin-top-5">
+        <div class="govuk-grid-column-full">
+            <header>
+                <h1 class="govuk-heading-l  govuk-!-margin-bottom-1">
+                    Manage deputy contact details
+                </h1>
+            </header>
+
+            <div class="govuk-grid-row">
```

Comment:
Maybe one for the accessibility reviews with Jen and can't find it on the ticket, but I take it no aria tags were needed initially for this?

Reply:
It's a good question. Jen didn't mention anything and I took the code from the latest prototype. I'll follow up with Jen and if there's any needed, I can add them in a separate PR.